packages/application-shell/src/utils/get-mc-api-url/get-mc-api-url.ts

```diff
@@ -0,0 +1,19 @@
+import { ApplicationWindow } from '@commercetools-frontend/constants';
+
+declare let window: ApplicationWindow;
+
+const getMcApiUrlFromOrigin = () => {
```

Comment:
We should document a bit why we do this.

Reply:
Sure, will do! 👍